setup.py

```diff
@@ -6,6 +6,7 @@
 try:
     from setuptools import setup
 except ImportError:
+    # FIXME(peterbe): In what circumstances do you NOT have setuptools?
```

Comment:
This is part of the bootstrapping code generated by ``cookiecutter``. Many projects use it for compatibility because distutils is native python. That said i don't really mind removing this.